packages/volto/src/express-middleware/robotstxt.js

```diff
@@ -4,13 +4,12 @@ import { generateRobots } from '@plone/volto/helpers/Robots/Robots';
 /*
 robots.txt - priority order:
 
-1) robots.txt in /public folder
```

Comment:
Note to self: It wasn't obvious at first, but this is only updating the comment to match existing behavior, and not changing what the code does.